spec/factories/tasks.rb

```diff
@@ -1,5 +1,10 @@
 FactoryBot.define do
   factory :task do
-    
+    sequence(:title) { |n| "title_#{n}" }
```

Comment:
sequenceの第二引数で定義した文字にはループの度に`String#next`を末尾の文字に実行されます！
https://docs.ruby-lang.org/ja/latest/method/String/i/next.html
```suggestion
    sequence(:title, "title_1")
```

Reply:
c7cdad8